### ISSUE TITLE
Inline bots suggestion does not persist after update

### ISSUE THREAD

**Issue by SmartManoj** [telegramdesktop/tdesktop#6270]
### Steps to reproduce
1. Update Telegram
2. No inline bots suggestion
3. 

### Expected behaviour
Tell us what should happen

### Actual behaviour
Tell us what happens instead

### Configuration
**Operating system:** Windows 10

**Version of Telegram Desktop:** 1.7.14

**Used theme**:

<details><summary><b>Logs</b>:</summary>
Insert log.txt here (if necessary)
</details>

**Comment by 23rd** [comment]
What bot did you use?
I tried `@youtube` and suggestions work.

Please, can you provide more information text in the future? If you continue to create issues with 5 words without any explanation, I will be forced to close them with `invalid` label.
I can't ask a bunch of follow-up questions on every your issue to find out what exactly you're asking.

**Comment by SmartManoj** [comment]
![image](https://user-images.githubusercontent.com/7231077/61038008-da7f6400-a3e9-11e9-88f7-0fe5ab08ac74.png)
This suggestion

I thought , title are self-explanatory.

**Comment by 23rd** [comment]
Recently used inline bots and hashtags are local data. If you lose your local data, your suggestions will be empty.

**Comment by SmartManoj** [comment]
It happens after update

**Comment by 23rd** [comment]
I updated from 1.7.13 to 1.7.14 and my suggestions are still working.

**Comment by Aokromes** [comment]
works for me (TM)

**Comment by Aokromes** [comment]
no answer after near 1 month, if this bug is still valid, open a new ticket with one updated telegram version downloaded from desktop.telegram.org or github.com

**Comment by SmartManoj** [comment]
> works for me (TM)

TM?

**Comment by AlexWayfer** [comment]
> TM?

Look at the first result of Google Search, please.

---

@Aokromes, please re-open. I have issues with inline mode after some update a few months ago. I don't know how to debug. Interesting, but it works in TDesktop on Linux and stops working in TDesktop on Windows. And I have a friend for which inline-bots don't work neither on Windows, nor on Linux, only in mobile applications.

Screenshots from Linux (Arch), version 2.1.13.

![image](https://user-images.githubusercontent.com/6510020/88371530-e04daa80-cd9c-11ea-9255-0c353a8b3fd5.png)

![image](https://user-images.githubusercontent.com/6510020/88371556-ecd20300-cd9c-11ea-9213-a4aacea9061c.png)

I'll send screenshots and version from Windows later.

There is definitely a some problem.

**Comment by SmartManoj** [comment]
> > TM?
> 
> Look at the first result of Google Search, please.
> 
>
trademark?

**Comment by SmartManoj** [comment]
@AlexWayfer still have this issue?

**Comment by AlexWayfer** [comment]
> > > TM?
> > 
> > 
> > Look at the first result of Google Search, please.
> 
> trademark?

Yes (however, it's a little inappropriate).



> @AlexWayfer still have this issue?

Yes, unfortunately. But I found a kind of work-around: you have to write full bot nickname manually, and it'll work instantly, and only after this (first) time Telegram will cache its name as inline bot.

@Aokromes can we improve this behavior somehow?
